Gemfile

```diff
@@ -2,3 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in action_subscriber.gemspec
 gemspec
+
+if RUBY_ENGINE == "ruby" && RUBY_VERSION.split(".").first.to_i < 2
```

Comment:
@liveh2o I found out that the whole requirement for ruby 2.0+ only exists for MRI. Under jRuby there is no requirement on the `amq-protocol` gem, because `march_hare` includes its own implementation of the rabbitmq protocol.

We could definitely pull this out of this branch and put it into master with a patch release. That would resolve our 1.9 support issues.